website/content/docs/commands/init.mdx

```diff
@@ -66,11 +66,20 @@ string](/docs/templates/hcl_templates/blocks/packer#version-constraints). The
 plugins will be installed in the [Plugin
 Directory](/docs/configure#packer-s-plugin-directory).
 
+### Implicit required plugin
+
+Starting from Version `1.8` many plugins will move out from Packer core. This is
+part of a set of breaking changes made to decouple Packer releases from a Plugin
+release. To make the transition easier we will tag components of these plugins
```

Comment:
easier to make a whole-paragraph suggestion than line by line, so here goes: 

Starting from Version `1.8` many plugins will move out from the Packer core. 
This is part of a set of breaking changes made to decouple Packer releases from 
plugin releases. To make the transition easier, we will tag components of these 
plugins as "moved out". If one of the components of a moved out plugin is used 
in a config file, but there is no mention of that plugin in the 
"required_plugin" block, then Packer init will automatically download and 
install that plugin. Packer will then display a warning and suggest that you
add the plugin to your `required_plugin` block. We recommend you use the 
`required_plugin` block even if you are only using official plugins, because it
allows you to set the plugin version to avoid surprises in the future.

Reply:
Thanks !